frontend/src/bundles/studio/store/slice.ts

```diff
@@ -79,6 +89,10 @@ const initialState: State = {
     scenes: [{ id: uuidv4(), duration: MIN_SCENE_DURATION }],
     scripts: [],
     videoSize: VideoPreview.LANDSCAPE,
+    voices: {
+        dataStatus: DataStatus.IDLE,
```

Comment:
We should have only one dataStatus for slice like it was mentioned in discussion https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/236#discussion_r1751960832